src/tracker.h

```diff
@@ -30,13 +30,46 @@
 #ifndef SSR_TRACKER_H
 #define SSR_TRACKER_H
 
+#include <atomic>
+#include <thread>
+
+#include "geometry.h"
+
+
+namespace ssr
+{
+
 /// Class definition
-struct Tracker
+class Tracker
 {
-  virtual ~Tracker() = default;  ///< destructor
+  public:
+    Tracker(api::Publisher& controller) : _controller(controller){};  ///< constructor
 
-  /// calibrate tracker; set the instantaneous position to be the reference
-  virtual void calibrate() = 0;
+    virtual ~Tracker() = default;  ///< destructor
+
+    /// reset tracker; set the instantaneous position to be the reference
+    virtual void reset()
+    {
+      SSR_VERBOSE2("Tracker reset.");
+      this->_correction_rot = this->_current_rot;
+    }
+
+    // Update SSR
+    virtual void update()
+    {
+      ssr::quat r = inverse(_correction_rot) * inverse(_current_rot);
```

Comment:
Why do you call `inverse()` twice?

Shouldn't it be enough to call `inverse()` once in `reset()` and here only left-multiply with the already inversed value?